cmd/registry/patterns/resources.go

```diff
@@ -24,6 +24,8 @@ import (
 	"github.com/apigee/registry/server/registry/names"
 )
 
+const ResourceUpdateThresholdSeconds = 2
```

Comment:
Maybe update this to a duration: `var ResourceUpdateThreshold = 2 * time.Second`. Then instead of converting to seconds before comparing, you can use the `time` package `Before` and `After` methods.

Reply:
Done.